--- a/net/minecraft/game/world/path/Pathfinder.java
+++ b/net/minecraft/game/world/path/Pathfinder.java
@@ -61,7 +61,7 @@
 			}
 
 			pathPoint25.isFirst = true;
-			Object object15 = 0;
+			int object15 = 0;
 			byte b16 = 0;
 			if(pathfinder20.getVerticalOffset(pathPoint25.xCoord, pathPoint25.yCoord + 1, pathPoint25.zCoord, pathPoint5) > 0) {
 				b16 = 1;
@@ -90,7 +90,7 @@
 
 			object9 = object15;
 
-			for(int i26 = 0; i26 < object9; ++i26) {
+			for(int i26 = 0; i26 < object15; ++i26) {
 				PathPoint pathPoint11 = pathfinder20.pathOptions[i26];
 				float f12 = pathPoint25.totalPathDistance + pathPoint25.distanceTo(pathPoint11);
 				if(!pathPoint11.isAssigned() || f12 < pathPoint11.totalPathDistance) {
